### PR TITLE
Documentation: Update instruction for building with QtCreator

### PR DESCRIPTION
**Pull request by guerinoni**

### CODE REVIEW

File: Documentation/QtCreatorConfiguration.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ First, make sure you have a working toolchain and can build and run SerenityOS.
 * Wait for the file list to generate. This can take a minute or two!
 * Ignore the file list, we will overwrite it later. Click Next.
 * Set `Add to version control` to `<None>`. Click Finish.
-* In your shell, go to your SerenityOS project directory, and invoke the `Meta/refresh-serenity-qtcreator.sh` script to regenerate the `serenity.files` file. You will also have to do this every time you delete or add a new file to the project.
```
Comment (AtkinsSJ) [inline Documentation/QtCreatorConfiguration.md:14]: "SerenityOS project directory" needs changing too

Comment (guerinoni) [reply Documentation/QtCreatorConfiguration.md:14]: Can I do in a different PR?

Comment (AtkinsSJ) [reply Documentation/QtCreatorConfiguration.md:14]: I guess you'll have to now. :sweat_smile: Andrew was too fast for me

Comment (ADKaster) [reply Documentation/QtCreatorConfiguration.md:14]: whoops :sweat_smile:

Comment (guerinoni) [reply Documentation/QtCreatorConfiguration.md:14]: No problem, i'll throw a new one :)

**Pull request merged**